requirements.txt

```diff
@@ -3,7 +3,7 @@ matplotlib>=2.1
 networkx>=2.0
 numpy>=1.13
 ply>=3.10
-scipy>=0.19
+scipy>=0.19,!=0.19.1
```

Comment:
What's the context for this? should we just bump up the dependency?

Reply:
Might be related to #829? This line does look a bit odd indeed.

Reply:
For some reason scipy 19.1 doesn't implement `unitary_group` although it does before and after this.